### ISSUE TITLE
A simple sound cue on prompt finish

### ISSUE THREAD

**Issue by bastetfurry** [ollama-webui/ollama-webui#593]
**Is your feature request related to a problem? Please describe.**
As i run the server on a small home server it is CPU bound. I can live perfectly fine with the speed, or lack thereof, but it would be cool if the WebUI could tell me that a prompt is finished trough a bell or other sound so that i can let it run in the background and come back to it later.

**Describe the solution you'd like**
A sound being played when a prompt has been finished.

**Describe alternatives you've considered**
None, playing a sound should be a relative simple quality of life feature.

**Additional context**
Just have some "ping.wav" as the default and when the user doesn't like it they can upload or simply replace the default with their favorite.
Definitely not a priority but a nice to have. :)

**Comment by justinh-rahb** [comment]
Ollama WebUI already supports OS notifications upon generation completion. I believe it's up to your OS whether these will play a sound.

**Comment by bastetfurry** [comment]
Then somehow Firefox missed the cue to enable them. I will have a look, thanks.

**Comment by justinh-rahb** [comment]
Afraid I'm not familiar with Firefox anymore, but in Chrome it prompts for permission the first time it attempts to send a notification, or it can be enabled in the "site settings" for the domain you've got your UI hosted on.

**Comment by tjbck** [comment]
Unable to reproduce the issue with all three of my machines (ubuntu, macos, windows). You might want to check if firefox notification is blocked on a OS level, Keep us updated!

**Comment by NightMachinery** [comment]
How do I enable this notification? 

Adding a way to play a custom sound is much better though. The OS notification sounds are usually intrusive.
